Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla43941.cs

```diff
@@ -85,7 +83,7 @@ public LandingPage43941()
 				HorizontalTextAlignment = TextAlignment.Center,
 				VerticalTextAlignment = TextAlignment.Center
 			};
-
+			var childPages = new List<ContentPage43941>();
```

Comment:
What is the motivation for storing the pages in the `childPages` list?

Reply:
If we lose link  on ```ContentPage43941``` then Garbage collector can invoke finalizer of ```ContentPage43941``` at any time. Therefore test behaviour can be different each run. Sometimes counter can be zero after test ```for``` loop.

P.S. This changes are out of issue scope. I just want to remove comment ```// At this point, the counter can be any value, but it's most likely not zero.```

Reply:
The original issue this test was written for (https://bugzilla.xamarin.com/show_bug.cgi?id=43941) was a memory leak of the ContentPage. The problem was that the counter could _never_ get to zero. 

As long as the instance count is zero at the end of the test, we don't care what the counter values were in the interim. So there's no reason to keep the list around to prevent garbage collection; garbage collection is the thing we _want_ to happen.

Reply:
I understand that garbage collection is thing we want to happen. Therefore before ```GC.Collect()``` link on ```childPages``` will be removed. Common idea is prevent invoking finalizers at a random time. Finalizers will be started only after _GC button_ click.